### DISCUSSION TITLE
Can you customize the indentation style

### DISCUSSION THREAD

**Discussion by lhjok** [brotzeit/rustic#494] [Q&A]
Chain access and square brackets of macros have strange indentation

**Comment by brotzeit** [comment]
Does `rust-indent-offset` work ?

**Reply by lhjok** [reply]
I want to wrap lines after brackets and square brackets without indentation, instead of setting how many spaces.

**Reply by lhjok** [reply]
Sometimes we need chain calls, and the code will look strange after indentation.

**Reply by lhjok** [reply]
When the first letter is a right bracket or a right square bracket, the line break does not need to be indented.

**Reply by lhjok** [reply]
Already solved `(setq rustic-indent-method-chain t) `
